nncf/common/graph/patterns/patterns.py

```diff
@@ -281,6 +281,9 @@ class PatternNames(Enum):
     and would not be quantized in compression pipeline.
     """
 
+    # ATOMIC OPERATIONS
+    L2_NORM = PatternDesc('l2_norm')
```

Comment:
We already have NORMALIZE_L2_MULTIPLY. What is the diff between them?

Reply:
I'll check it

Reply:
It's differs from Openvino `NORMALIZE_L2_MULTIPLY`